### DISCUSSION TITLE
Very high temperature with vram

### DISCUSSION THREAD

**Discussion by KarimGhazlaoui** [NVlabs/instant-ngp#374] [Q&A]
Hello,

I'm using a 3090 Fe, my vram reach 110° without training a model. ramping up slowly but in like two minutes it will reach max temp and the GPU will start limit it's performance.
Any idea how i could fix this issue ? 

Thanks.

![NGP](https://user-images.githubusercontent.com/51721465/156144227-51bc8659-adde-457c-8c85-489e88d0a126.png)

**Comment by half-potato** [comment]
Bro fix your computer
